dev-tools/build.sh

```diff
@@ -8,15 +8,15 @@ composer config platform.php 2> /dev/null || composer config platform.php 5.3.6
 composer require --no-update symfony/polyfill-mbstring
 
 # install package deps without dev-deps / remove already installed dev-deps
-composer update --no-interaction --no-progress --no-dev --prefer-stable
+composer update --no-interaction --no-progress --prefer-stable
```

Comment:
why installing with devs ?

Reply:
If you let Box find out the files for you, it will automatically remover the dev related stuff, so you don't have to care about it and it avoids having to restore the dependencies behind